resources/definitions/bel_all_metal_delta.def.json

```
@@ -0,0 +1,94 @@
+        "machine_extruder_count":          { "default_value": 1, "maximum_value": "4" },
+        "machine_heated_bed":              { "default_value": true },
+        "machine_center_is_zero":          { "default_value": true },
+        "machine_gcode_flavor":            { "default_value": "RepRap (RepRap)" },
+        "machine_start_gcode":             { "default_value": ";---------------------------------------\n;Metal Delta start script\n;---------------------------------------\nT0\nT1\nG28 ;Home\nG1 Z15.0 F2000 ;Move the platform down 15mm\nG1 Y80 F2000\nG1 Z10 F200\nG92 E0\nG1 E3 F100\nG92 E0\n;---------------------------------------"},
+        "machine_end_gcode":               { "default_value": ";---------------------------------------\n;Metal Delta end script\n;---------------------------------------\nM104 S0\nM140 S0\n;Retract the filament\nG92 E1\nG1 E-1 F300\nG28 X0 Y0\nM84\n" },
+        "machine_shape":                   { "default_value": "elliptic" },
+
+        "machine_max_feedrate_z" :         { "default_value": 300 },
+
+        "speed_print" :                    { "default_value": 45 },
+        "speed_topbottom":                 { "value": "speed_print * 0.90" },
+        "speed_wall_0":                    { "value": "35 if speed_print > 35 else speed_print" },
+        "speed_layer_0":                   { "value": "speed_print * 0.55" }, 
+
+        "speed_travel":                    { "value": "100" },
+        "speed_travel_layer_0":            { "value": "speed_travel * 0.70" },
+        "speed_z_hop":                     { "value": "speed_travel" },
+
+        "acceleration_enabled":            { "value": "false" },
+        "acceleration_travel" :            { "value": "9000" },
+        "acceleration_print":              { "value": "4000" },
+        "acceleration_wall":               { "value": "acceleration_print * 0.5" },
+        "acceleration_wall_0":             { "value": "acceleration_wall * 0.5" },
+        "acceleration_topbottom":          { "value": "acceleration_wall_0" },
+        "acceleration_layer_0":            { "value": "acceleration_wall_0" },
+        "acceleration_prime_tower":        { "value": "acceleration_wall" },
+        "acceleration_support":            { "value": "acceleration_wall" },
+        "acceleration_support_interface":  { "value": "acceleration_wall_0" },
+
+        "jerk_enabled":                    { "value": "True" },
+        "jerk_print":                      { "value": "5" },
+        "jerk_infill":                     { "value": "5" },
+        "jerk_travel":                     { "value": "5" },
+
+        "retraction_hop_enabled":          { "default_value": false },
+        "retraction_hop":                  { "default_value": 0.5 },
+        "retraction_amount" :              { "default_value": 3.5 },
+        "retraction_speed" :               { "default_value": 70 },
+        "retraction_combing" :             { "default_value": "noskin" },
```

This is causing the tests to fail. This setting value would not be honoured because it's overridden by a formula in a `value` property. To properly override the setting:

```suggestion
        "retraction_combing" :             { "value": "'noskin'" },
```

---

Thank you for taking your time to look through it! I had not found the time to really go through the error message yet!